src/components/business-lookup/BusinessLookup.vue

```diff
@@ -154,6 +154,9 @@ export default class BusinessLookup extends Vue {
   /** Business status to search for (eg, ACTIVE or HISTORICAL or '' to match all. */
   @Prop({ default: 'ACTIVE' }) readonly searchStatus!: string
 
+  /** Label for BusinessLookup component. */
+  @Prop({ default: 'Business or Corporation Name or Incorporation Number' }) readonly defaultLabel!: string
```

Comment:
This isn't the "default label" if you pass a different value as a prop. I would simply rename this to `label`.

Otherwise, everything here looks fine.